builders/scripts/builder_darwin_mac32_wine

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
```

Comment:
What is the difference between mac32 and i386?

Reply:
mac32 is what was previously mac64, so using the clang-7 trunking feature to build a 32bit version of wine that should function on pure 64bit macOS versions

Reply:
Ok makes sense. It does not work yet, does it?

Reply:
I didn't test but I thought it would be best to separate the 32bit builder from the expanded 64bit builder to facilitate faster testing later.
So mac64 would be a full 64Bit with wow64, mac32 would be just wine.